lib/ansible/modules/storage/netapp/na_cdot_snapshot.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+
+# (c) 2017, NetApp, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+
+module: na_cdot_snapshot
+
+short_description: Manage NetApp cDOT snapshots
+extends_documentation_fragment:
+    - netapp.ontap
+version_added: '2.5'
+author: Marcel Juhnke <marcel.juhnke@accenture.com>
+
+description:
+- Create or delete snapshots on NetApp cDOT
+
+options:
+
+  state:
+    description:
+    - Whether the specified snapshot should exist or not.
+    required: true
+    choices: ['present', 'absent']
+
+  name:
+    description:
+    - The name of the snapshot to manage.
+    required: true
+
+  volume_name:
+    description:
+    - The name of the containing volume.
+    required: true
+
+  vserver:
+    description:
+    - Name of the vserver to use.
+    required: true
+    default: None
+
+'''
+
+EXAMPLES = """
+
+    - name: Create Snapshot Copy
+      na_cdot_snapshot:
+        state: present
+        name: snap-keep-forever
+        volume_name: vserver_root
+        vserver: ansibleVServer
+        hostname: "{{ netapp_hostname }}"
+        username: "{{ netapp_username }}"
+        password: "{{ netapp_password }}"
+
+    - name: Delete Snapshot Copy
+      na_cdot_snapshot:
+        state: absent
+        name: snap-keep-forever
+        volume_name: vserver_root
+        vserver: ansibleVServer
+        hostname: "{{ netapp_hostname }}"
+        username: "{{ netapp_username }}"
+        password: "{{ netapp_password }}"
+
+"""
+
+RETURN = """
+
+
+"""
+import traceback
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_native
+import ansible.module_utils.netapp as netapp_utils
+
+
+HAS_NETAPP_LIB = netapp_utils.has_netapp_lib()
+
+
+class NetAppCDOTSnapshot(object):
+
+    def __init__(self):
+
+        self._size_unit_map = dict(
```

Comment:
This isn't necessary, since it isn't used in this module

Reply:
True, I will remove it. Was an unnoticed leftover from when I copied the class initially from the na_cdot_volume module.